airgun/views/ansible_role.py

```diff
@@ -21,10 +21,10 @@ class AnsibleRolesView(BaseLoggedInView, SearchableViewMixin):
     is present, without the search widget or table.
     """
 
-    title = Text("//h1")
+    title = Text("//h1[text()]")
```

Comment:
why are using the `text()` here ?

Reply:
Just to be a little precise I was matching the exact string but I think the tag itself will do the job please correct me here @omkarkhatavkar

Reply:
I don't this it is required there

Reply:
updated